files/galaxy/dynamic_job_rules/production/total_perspective_vortex/users.yml

```diff
@@ -115,6 +115,17 @@ users:
       scheduling:
         require:
           - slurm
+      rules:
+        - id: star_minio_test_rule
+          if: |
+            'fastqc' in tool.id
+          params:
+            object_store_id: minio_test
+        - id: star_aws_gateway_test_rule
+          if: |
+            'samtools_merge' in tool.id
```

Comment:
I was originally thinking that we would have two different users - one who would be exclusively routed to minio and the other to aws gateway. Was there a specific reason to choose this configuration?

Reply:
yeah, I didn't have their email addresses so I thought I would use a test user as a first pass and add them later